### ISSUE TITLE
Don't take on too much!

### ISSUE THREAD

**Issue by johnaldmcfist** [brainhack-school2020/fibers-fMRI-BHS2020#1]
You described a lot of techniques you wish to learn. Be mindful of the time it can take to properly learn these. I suggest you focus on the tools and use fMRI or dMRI as a medium to learn them :)

**Comment by SamGuay** [comment]
I must agree with @johnaldmcfist ! You may want to consider focusing on only one of the projects you described. At least, I would suggest doing one project from start to finish. Then, if things go smoothly (which is not often the case for me :sweat_smile:), you *may* end up with some free time to start your second project.

**Comment by guenounz** [comment]
thank you both !
